Kubernetes Installer v2023.06.09-0 release notes (#1213)
replicatedhq/replicated-docs: docs/release-notes/rn-kubernetes-installer.md

<!--RELEASE_NOTES_PLACEHOLDER-->

## v2023.06.09-0

Released on June 9, 2023

### New Features {#new-features-v2023-06-09-0}
* Adds [Prometheus add-on](https://kurl.sh/docs/add-ons/prometheus) version 0.65.1-46.5.0.
* Adds [EKCO add-on](https://kurl.sh/docs/add-ons/ekco) version 0.27.1.
* Adds [Rook add-on](https://kurl.sh/docs/add-ons/rook) verison 1.11.7.
* Adds [OpenEBS add-on](https://kurl.sh/docs/add-ons/openebs) version 3.7.0.
* Adds [Prometheus add-on](https://kurl.sh/docs/add-ons/prometheus) version 0.65.1-46.6.0.

### Improvements {#improvements-v2023-06-09-0}

> * Enhanced to utilize the HTTP_PROXY and HTTPS_PROXY environment variables during the configuration of the container runtime.

Reviewer: ```suggestion
* Enhance proxy installs by enabling the use of HTTP_PROXY and HTTPS_PROXY environment variables during the configuration of the container runtime.
```

* Increase the Flannel waiting time to check that it is available and improve log in case of failures. This change are valid only for Flannel versions equals and upper than 0.21.5.

### Bug Fixes {#bug-fixes-v2023-06-09-0}

> * Fixes issue faced when migrating from weave to flannel where pods get stuck and ensure that weave is properly removed. This bug fix was introduced for Flannel versions equals and upper than 0.21.5.

Reviewer: ```suggestion
* Fixes issue when migrating from Weave to Flannel where pods get stuck. Moreover, the Weave to Flannel migration now ensures that Weave is properly removed. This fix applies to Flannel versions 0.21.5 and later.
```

<!--RELEASE_NOTES_PLACEHOLDER-->

## v2023.06.09-0

Released on June 9, 2023

### New Features {#new-features-v2023-06-09-0}
* Adds [Prometheus add-on](https://kurl.sh/docs/add-ons/prometheus) version 0.65.1-46.5.0.
* Adds [EKCO add-on](https://kurl.sh/docs/add-ons/ekco) version 0.27.1.
* Adds [Rook add-on](https://kurl.sh/docs/add-ons/rook) verison 1.11.7.
* Adds [OpenEBS add-on](https://kurl.sh/docs/add-ons/openebs) version 3.7.0.
* Adds [Prometheus add-on](https://kurl.sh/docs/add-ons/prometheus) version 0.65.1-46.6.0.

### Bug Fixes {#bug-fixes-v2023-06-09-0}

> * Fixes issue where script get stuck to delete Pods when it is migrating from weave to flannel. Bug fixed addressed only to 0.21.5.

Reviewer 2: We update the base so I will close this ones but would be great get your re-check.

<!--RELEASE_NOTES_PLACEHOLDER-->

## v2023.06.09-0

Released on June 9, 2023

:::important

> Previously releases has a known issue that can lead the system to a critical state when you are upgrading from Weave to Flannel. This known issue is resolved v2023.06.09-0. To avoid this known issue, do not upgrade with configurations to migrate from Weave to Flannel using previous releases and Flannel versions which are lower than 0.21.5. Note that the bug fixes were not backport to Flannel versions lower than 0.21.5. 

Reviewer 2: ```suggestion
Previously releases has a known issue that can lead the system to a critical state when you are upgrading from Weave to Flannel. This known issue is resolved v2023.06.09-0. To avoid this known issue, do not upgrade with configurations to migrate from Weave to Flannel using previous releases and Flannel versions which are lower than 0.21.5. Note that the bug fixes were not backport so far to Flannel versions lower than 0.21.5. 
```

Reviewer 2: Remove in favor to discuss it in the next week.

> :::important
> Previously releases has a known issue that can lead the system to a critical state when you are upgrading from Weave to Flannel. This known issue is resolved v2023.06.09-0. To avoid this known issue, do not upgrade with configurations to migrate from Weave to Flannel using previous releases and Flannel versions which are lower than 0.21.5. Note that the bug fixes were not backport to Flannel versions lower than 0.21.5. 
> :::

Reviewer 2: ```suggestion
```
